keyboards/durgod/k320/config.h

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2020 kuenhlee and Don Kjer
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0320
+#define DEVICE_VER      0x0003
+#define MANUFACTURER    Hoksi Technology
```

Comment:
https://durgod.com lists `Zhuhai Hoksi Technology` in all their page footers.

Reply:
I believe `Hoksi Technology` is what appears when you plug in the stock keyboard.  Do you have one around you can test and verify with?  I don't have a strong preference either way.

Should we change the VENDOR_ID to something else as well, to ensure we have a clean namespace for PRODUCT_ID?  I was thinking of using the last 3 digits of PRODUCT_ID for model, and the first digit for variant (e.g. no-backlight `0320`, backlight `1320`, rgb `2320`, etc..)  Or maybe the other way around looks better, `3200`, `3201`, etc?

Reply:
Unfortunately, I just ordered my first non-backlit Durgod K320 and non-backlit K310 today and they haven't shipped yet.  I'll have to give you a confirmation of what shows up in `lsusb` later.

I like the idea to show the other backlit variants.  Do you know anyone who owns any of them so you can find out if they use the same matrix perhaps?

I haven't looked yet at most of the other keyboards, but I suspect most/all of them are probably using the same `VENDOR_ID`.  It might be worth confining all the model/variant info to just the `PRODUCT_ID` and/or `DEVICE_VER` values in case it turns out that an "official" USB vendor ID is needed in some cases.

Reply:
Checking my notes, I believe this was the origin `lsusb` output for the stock K320 keyboard:
```
DURGOD Taurus K320:

  Product ID: 0x0082
  Vendor ID: 0x2f68
  Version: 1.04
  Speed: Up to 12 Mb/s
  Manufacturer: Hoksi Technology
  Location ID: 0x14500000 / 50
  Current Available (mA): 500
  Current Required (mA): 500
  Extra Operating Current (mA): 0
```
The original Vendor ID is `0x2f68`, however as drashna pointed out earlier in the PR using the OEM Vendor ID may be problematic.  I switched it to `0xFEED` since I saw a lot of other qmk keyboards using that, however I just changed it to `0xD60D` to give ourselves a clean Product ID namespace.

I also updated the Product ID to `0x3200` in preparation for backlight variants.

Reply:
I have a Durgod Hades which uses the very same `STM32F070RBT6` chip and the output of `lsusb` is:
```
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 
  bDeviceSubClass         0 
  bDeviceProtocol         0 
  bMaxPacketSize0        64
  idVendor           0x2f68 
  idProduct          0x4001 
  bcdDevice            1.03
  iManufacturer           1 Hoksi Technology
  iProduct                2 DGK68C
  iSerial                 0 
  bNumConfigurations      1
```

Reply:
Thanks ckoehn, good to know!